projects/nae-example-app/src/app/doc/public-single-task-view/public-single-task-view.component.ts

```diff
@@ -119,16 +124,17 @@ const caseResourceServiceFactory = (userService: UserService, sessionService: Se
         TaskRequestStateService,
         TaskEventService,
         {provide: NAE_TASK_OPERATIONS, useClass: SubjectTaskOperations},
+        {provide: AllowedNetsServiceFactory, useClass: AllowedNetsServiceFactory},
     ]
 })
-export class PublicSingleTaskViewComponent extends AbstractSingleTaskViewComponent implements AfterViewInit {
+export class PublicSingleTaskViewComponent extends AbstractSingleTaskViewComponent implements OnInit, AfterViewInit {
```

Comment:
changes were made to a component in the example-app. Does a schematic for generating these components exist? If yes, then its template files should be changed as well.

Reply:
Schematics will be resolved in standalone tasks. Thanks for the reminder, I will resolve all management regarded tasks tomorrow (PR correction, task creation into Jira, etc).